codeforamerica/vita-min
# Fyst 996 add two income sub md

## Link to pivotal/JIRA issue
https://codeforamerica.atlassian.net/browse/FYST-996
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

## What was done?
- add a two income subtractions form to maryland intakes
- add columns to capture primary and spouse deduction amounts
## How to test?
- go through the maryland flow for creating an intake
- when selecting a payload to import, select one with a fed student loan amount, like "zeuse many w2's"
- Verify
- Risk Assessment
- n/a
## Screenshots (for visual changes)
<img width="681" alt="Screenshot 2024-11-04 at 4 16 18 PM" src="https://github.com/user-attachments/assets/6fcc43d4-ea6b-4709-a11c-72e5cf779338">
